### PR TITLE
July 2023 newsletter - open for ideas

### PR DESCRIPTION
**Pull request by benjaoming**
* [x] Add all roadmap items deployed
* [x] Security advisories
* [x] Upcoming changes
* [x] Awesome project of the month
* [x] Possible issues
* [ ] Attractive screenshot for link previews

NOTE:

* We will announce via tweet / toot template, not manually facilitated:
  
  > 🗞️ Read our June newsletter: https://blog.readthedocs.com/newsletter-june-2023/

* @ericholscher will use Buffer to announce

### CODE REVIEW

**Comment by benjaoming** [comment]
I'm missing an idea for a link preview screenshot?

This screenshot makes it easier for us to share the newsletter with a standard text, which is our intention onward.

**Comment by humitos** [comment]
I assigned this PR to you @ericholscher. I think it has a good overview of what has happened in the last month, but there are some copy that you may want to adjust a little.

**Comment by ericholscher** [comment]
I've updated the copy here, and it should be ready to go if anyone can give it a 👍

File: newsletter-july-2023.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  See the documentation of the `build.tools <https://docs.readthedocs.io/en/latest/config-file/v2.html#build-tools>`__ for more info.
+- 📚️ A new documentation how-to for ``.readthedocs.yaml`` was started at `Write the Docs <https://www.writethedocs.org/>`__ and finished recently.
+  Read it here: :doc:`readthedocs:guides/setup/configuration-file`.
+- ⏩️ HTTP speedups: Several HTTP endpoints and CloudFlare configurations have been tweaked and are performing better.
+- 🐛️ Bug fix: We are now 100% relying on search indexing by parsing HTML, instead of special Sphinx-only logic. This makes search a lot simpler and more consistent for our users.
+
+Upcoming features
+-----------------
+
+- ⏩️ Our build process is receiving considerable improvements for larger Git repositories.
```
Comment (stsewd) [inline newsletter-july-2023.rst:32]:
```suggestion
- ⏩️ Our build process is receiving considerable improvements for large Git repositories.
```

Not sure if my suggestion is correct :D

File: newsletter-july-2023.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  The ``git clone`` and ``git fetch`` operations are tweaked,
+  which results in significant speedups.
+  For instance, the Cpython project now fetches its Git repository at nearly 10x the speed of before.
```
Comment (stsewd) [inline newsletter-july-2023.rst:35]:
```suggestion
  For instance, the CPython project now fetches its Git repository at nearly 10x the speed of before.
```

**Review by stsewd — APPROVED** [review]

File: newsletter-july-2023.rst (hunk 1 above)
Comment (ericholscher) [reply newsletter-july-2023.rst:32]: They both work :)

File: newsletter-july-2023.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
+.. post:: July 6, 2023
+   :tags: newsletter, python
+   :author: Eric
+   :location: BND
+   :category: Newsletter
+
+Read the Docs newsletter - July 2023
+====================================
+
+News and updates
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply newsletter-july-2023.rst:10]: Definitely seems worth mentioning! I'll include it.

[the remark this answers is not in the document]
Comment (ericholscher) [reply newsletter-july-2023.rst:10]: Updated 👍

**Pull request merged**